package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@openhpi/xikolo-video-player",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
Oh, did we forget to update this for the latest release? Can you please apply this change directly to `dev`? 🙏